dbt_subprojects/dex/models/trades/dex_base_trades.sql

```diff
@@ -46,6 +46,7 @@
     , ref('dex_zksync_base_trades')
     , ref('dex_zora_base_trades')
     , ref('dex_plume_base_trades')
+    , ref('dex_taiko_base_trades')
```

Comment:
small nit: looks like starting with plume one line above, we stopped the alphabetical order. can you move those two into alphabetical order?